pages/about.html

```diff
@@ -4,9 +4,33 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="/Users/miay/Developer/unit2-projects/personal-portfolio-site/styles/about.css" rel="stylesheet">
+  <title>About</title>
 </head>
 <body>
-  
+  <header>
+    <h1>About</h1>
+
+  </header>
+  <section>
+    <h2>Background</h2>
+    <p>Before joining Ada Developers Academy, Mia worked as a psychiatric nurse and was studying to become a mental health nurse practitioner. However, they realized that the lack of access to primacy mental health care is causing frequent relapse and hospitalization. Inspired by <a href="https://en.wikipedia.org/wiki/Thomas_R._Insel">Dr. Tom Insel</a> 's work in psychiatric research through modern technology, they made a transition into tech field hoping to increase accessibility of care for people in need.</p>
+  </section>
+  <section>
+    <h2>Interests</h2>
+    <p>Besides computer science and mental health, Mia also has a passion in music and performance art.</p>
+  </section>
 </body>
+<footer>
+  <nav>
+    <ul>
+      <li>
+        <a href="/Users/miay/Developer/unit2-projects/personal-portfolio-site/index.html">Home</a>
+      </li>
+      <li>
+        <a href="/Users/miay/Developer/unit2-projects/personal-portfolio-site/pages/portfolio.html">Portfolio</a>
```

Comment:
The portfolio page is in the same folder, so as a relative path, this would be `portfolio.html`